rln/src/public.rs

```diff
@@ -19,7 +19,8 @@ cfg_if! {
     if #[cfg(not(target_arch = "wasm32"))] {
         use std::default::Default;
         use std::sync::Mutex;
-        use crate::circuit::{circom_from_folder, vk_from_folder, circom_from_raw, zkey_from_folder, TEST_RESOURCES_FOLDER, TEST_TREE_HEIGHT};
+        use crate::circuit::{circom_from_folder, vk_from_folder, circom_from_raw, arkzkey_from_folder, TEST_RESOURCES_FOLDER, TEST_TREE_HEIGHT};
```

Comment:
perhaps there is a way we can maintain CI properly while merging this in, can we use arkzkey behind a feature flag? i.e behind the arkzkey feature flag we have the imports, loading logic etc, and in the default features we just continue to use zkey

Reply:
that sounds good, I'll add this one